README.md

```diff
@@ -189,6 +189,38 @@ Package names of arguments can be separated by spaces.
  setuptools  38.5.0   UNKNOWN
 ```
 
+### Option: packages
+
+When executed with the `--packages` option, only the packages specified by the argument are placed into the list output.
+
+```bash
+(venv) $ pip-licenses --packages twine numpy
+ Name  Version  License
+ numpy 1.16.1   BSD
+ twine 1.12.1   MIT
+```
```

Comment:
The implementation does not work as of this document.
```bash
# error with this option
$ pip-licenses --packages twine numpy
usage: piplicenses.py [-h] [-v] [--from SOURCE] [-s] [-a] [-u] [-d] [-l] [--no-license-path] [-i PKG] [-p INCLUDES] [-o COL]
                      [-f STYLE] [--summary] [--output-file OUTPUT_FILE]
piplicenses.py: error: unrecognized arguments: numpy
```

Since `action='append'` is used, it must be specified multiple times. Please see [docs for argpase](https://docs.python.org/3.8/library/argparse.html).

```bash
# work with this option
$ pip-licenses --packages twine --packages numpy
```

The behavior of the existing option `--ignore-packages` is also a breaking change.

Is it difficult to achieve your function with a combination of `action='store'` and `nargs='+'`?